addons/web/__manifest__.py

```diff
@@ -21,21 +20,541 @@
         'views/base_document_layout_views.xml',
         'data/report_layout.xml',
     ],
-    'qweb': [
-        "static/src/xml/base.xml",
-        "static/src/xml/chart.xml",
-        "static/src/xml/fields.xml",
-        "static/src/xml/file_upload_progress_bar.xml",
-        "static/src/xml/file_upload_progress_card.xml",
-        "static/src/xml/kanban.xml",
-        "static/src/xml/menu.xml",
-        "static/src/xml/notification.xml",
-        "static/src/xml/pivot.xml",
-        "static/src/xml/rainbow_man.xml",
-        "static/src/xml/report.xml",
-        "static/src/xml/search_panel.xml",
-        "static/src/xml/web_calendar.xml",
-        "static/src/xml/graph.xml",
-    ],
+    'assets': {
+        #----------------------------------------------------------------------
+        # MAIN BUNDLES
+        #----------------------------------------------------------------------
+        # These are the bundles meant to be called via "t-call-assets" in
+        # regular XML templates.
+        #
+        # The convention to name bundles is as following:
+        # 1) the module in which the bundle is first defined (check module
+        #    order according to their dependencies), followed by a dot,
+        # 2) an arbitrary name, relevant to the content of the bundle. Note
+        #    that there is no need to put the "assets" keyword in the bundle
```

Comment:
reference to old way to write assets is not necessary --> should be in message commit